src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -25,19 +25,29 @@ This component provides the user with details on what is missing, and
 can upload the files. Otherwise, it can provide a consent checkbox where the
 user acknowledges that they will have to mail or fax the missing documents.
 */
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import {
   VaCheckboxGroup,
   VaTelephone,
 } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import { titleUI } from 'platform/forms-system/src/js/web-component-patterns';
 import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
 import PropTypes from 'prop-types';
-import { getConditionalPages } from '../../utilities';
+import {
+  getConditionalPages,
+  addStyleToShadowDomOnPages,
```

Comment:
Using this helper to inject CSS into shadow dom of VaCheckboxGroup since the main .scss file won't affect it.